<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>270</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Electra - IDs &amp; API Keys</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Images/Icon.png</normaloff>../Images/Icon.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QTextEdit" name="InfoTextbox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>251</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="autoFormatting">
    <set>QTextEdit::AutoBulletList</set>
   </property>
   <property name="tabChangesFocus">
    <bool>true</bool>
   </property>
   <property name="lineWrapMode">
    <enum>QTextEdit::WidgetWidth</enum>
   </property>
   <property name="lineWrapColumnOrWidth">
    <number>0</number>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="tabStopWidth">
    <number>80</number>
   </property>
   <property name="acceptRichText">
    <bool>true</bool>
   </property>
   <property name="cursorWidth">
    <number>1</number>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
